### PR TITLE
WIP: Update to skywire M2.

### LINKED ISSUE #9: Update skybian to skywire v0.1.0

### PR DESCRIPTION
**Pull request by evanlinjin**
Fixes #9

Changes:
- TODO

Does this change need to mentioned in CHANGELOG.md?
> Yes

How to test this PR:
1. Clone this repo. 
2. Checkout the PR branch. 
3. Run `make run-skyimager-gui`
4. Navigate to http://127.0.0.1:8080/

### CODE REVIEW

File: build.conf [hunk 1]
```diff
@@ -39,29 +38,18 @@ ARMBIAN_IMG_7z=""
...
-SKYCOIN_DIR=${GOPATH}/src/github.com/SkycoinProject
-SKYWIRE_DIR=${SKYCOIN_DIR}/skywire
-SKYWIRE_GIT_URL="https://github.com/SkycoinProject/skywire.git"
 
-# Skywire IP settings
+# Skywire settings
 SKYMINER_GATEWAY="192.168.0.1"
-SKYMINER_MANAGER="192.168.0.2"
+SKYMINER_HYPERVISOR_PK=""
 SKYMINER_NODES="192.168.0.3 192.168.0.4 192.168.0.5 192.168.0.6 192.168.0.7 192.168.0.8 192.168.0.9"
```
Comment (jdknives) [inline build.conf]: These might not be necessary anymore, since we do not have communication over local network between nodes anymore.

**Comment by Asgaror** [comment]
The commands and path structure of Skybian does not work properly:
image

Also, could you please provide some insight:
- where the config files should be located?
- how Skyflash will take care of finishing the config files wrt. to the hypervisor public key?

**Comment by evanlinjin** [comment]
@Asgaror to answer your queries...

> The commands and path structure of Skybian does not work properly:

To fix this issue, all you need to run is `sudo update-command-not-found`. And I wasn't aware the `hypervisor` was needed in Skybian. I am in contact with @jdknives to sort this out.

> where the config files should be located?

Currently config and database files for `visor` and apps are located in the `/root` folder. This will be changed to os default locations.

> how Skyflash will take care of finishing the config files wrt. to the hypervisor public key?

Please refer to https://github.com/SkycoinProject/skybian/blob/654cbca3b11724aaed1d9f15b4084236eeedbcd4/static/skybian-conf

**Comment by jdknives** [comment]
@evanlinjin the hypervisor does not need to be included in the image actually. But the same error occured for `skywire-visor`, correct @Asgaror ?

**Comment by Asgaror** [comment]
@jdknives @evanlinjin Thank you, the error occurs only for the hypervisor. I wasn't aware that there were no plans to include it in the Skybian image.`skywire-visor` works just fine

**Pull request merged**